cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/SystemAppTask.java

```diff
@@ -126,7 +172,8 @@ private SystemAppTaskContext buildTaskSystemAppContext(Injector injector, String
     SecureStore secureStore = injector.getInstance(SecureStore.class);
     ArtifactManagerFactory artifactManagerFactory = injector.getInstance(ArtifactManagerFactory.class);
     RemoteClientFactory remoteClientFactory = injector.getInstance(RemoteClientFactory.class);
-    return new DefaultSystemAppTaskContext(cConf, preferencesFetcher, pluginFinder, secureStore, systemAppNamespace,
+    return new DefaultSystemAppTaskContext(injector.getInstance(CConfiguration.class),
```

Comment:
Is the cConf passed to this expected to be modified?

Reply:
It is no different than the old code. We generally not expect to modify the cConf.